src/scanners/binary.js

```diff
@@ -13,7 +13,9 @@ export default class BinaryScanner extends BaseScanner {
   }
 
   check(buffer, values) {
-    for (let v in values) {
+    const valuesKeys = Object.keys(values);
+    for (let i = 0; i < valuesKeys.length; i++) {
+      const v = valuesKeys[i];
```

Comment:
I'm going to switch this to use `Array.prototype.some`. It's purpose built for this use-case.